decidim-core/app/presenters/decidim/admin_log/attachment_collection_presenter.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Decidim
+  module AdminLog
+    # This class holds the logic to present a `Decidim::Component`
+    # for the `AdminLog` log.
+    #
+    # Usage should be automatic and you shouldn't need to call this class
+    # directly, but here's an example:
+    #
+    #    action_log = Decidim::ActionLog.last
+    #    view_helpers # => this comes from the views
+    #    ComponentPresenter.new(action_log, view_helpers).present
```

Comment:
```suggestion
    #    AttachmentCollectionPresenter.new(action_log, view_helpers).present
```
This comes from the original copied file, we should update it